koku/reporting/provider/gcp/models.py

```diff
@@ -0,0 +1,54 @@
+"""Models for GCP cost and usage entry tables."""
+from django.db import models
+
+from api.provider.models import Provider
+
+
+class GCPCostEntryBill(models.Model):
+    """The billing information for a Cost Usage Report.
+
+    The billing period (1 month) will cover many cost entries.
+
+    """
+
+    class Meta:
+        """Meta for GCPCostEntryBill."""
+
+        unique_together = ('billing_period_start', 'provider')
+
+    billing_period_start = models.DateTimeField(null=False)
+    billing_period_end = models.DateTimeField(null=False)
+    summary_data_creation_datetime = models.DateTimeField(null=True)
+    summary_data_updated_datetime = models.DateTimeField(null=True)
+    finalized_datetime = models.DateTimeField(null=True)
+    derived_cost_datetime = models.DateTimeField(null=True)
```

Comment:
We should consider adding `blank=True` here just in case we ever want to rely on form validation in Django.